src/libcommonserver/io/iohelper.cpp

```diff
@@ -430,10 +474,12 @@ bool IoHelper::getFileSize(const SyncPath &path, uint64_t &size, IoError &ioErro
         return false;
     }
 
-    const bool isSymlink = itemType.linkType == LinkType::Symlink;
-    if (isSymlink) {
+    if (itemType.linkType == LinkType::Symlink) {
```

Comment:
My personal feeling: a `switch` statement would be easier to read.

Reply:
OK